.gitignore

```diff
@@ -16,3 +16,4 @@ bin
 commercial_package
 .vscode
 .vale
+.idea/
```

Comment:
@CarmiWisemon Does this change need to be in this PR?

Reply:
@CarmiWisemon 
Actually, you don't have to add .idea to the gitignore file. It's already there in the openshift-docs repository

Reply:
@sbeskin-redhat It adds it by itself.

Reply:
@mburke5678 It has been removed and you can ignore it.

Reply:
@CarmiWisemon Can we get the .gitignore out of the PR?

Reply:
@shahsahil264 @mburke5678 Can we just delete it here?

Reply:
@mburke5678 I deleted the file from the PR

Reply:
I don't think that's what they meant.. deleting the file. They just meant you don't have to make the change. aka keep the file as it was.